zipline/data/us_equity_pricing.py

```diff
@@ -778,24 +795,17 @@ def last_available_dt(self):
 
     @property
     def trading_calendar(self):
-        return None
+        return self._trading_calendar
```

Comment:
Okay, to get around this we could just have `trading_calendar = None` in the class body and then in `__init__` have `self.trading_calendar = calendar`. I think either way is fine. If we leave the proprty as it is can we have a comment so people don't try to remove it later?